### ISSUE TITLE
How To Use openAnimation Property Properly

### ISSUE THREAD

**Issue by DPOLocalAaronO** [react-component/collapse#136]
The documentation is vague, I need to know how to customize the open/close animation and also call a function on open. Thank you

**Comment by arbiasgjoshi** [comment]
I agree, the documentation is very vague and it gives the impression that it was done very fast without any clear goal on mind. I had to go at the examples folder, and in the util see his own declaration on how to implement it, which is very misleading :/

**Comment by paule89123** [comment]
Did anyone manage to get the animation to work? I've wasted so much time trying to get it to work but was unsuccessful
